content/en/security/ai_guard/onboarding.md

```diff
@@ -0,0 +1,538 @@
+
+#### Example: Evaluate a tool call {#java-example-evaluate-tool-call}
+
+Like evaluating user prompts, the method can also be used to evaluate tool calls:
+
+```java
+import datadog.trace.api.aiguard.AIGuard;
+
+final AIGuard.Evaluation evaluation = AIGuard.evaluate(
+    Collections.singletonList(
+        AIGuard.Message.assistant(
+            AIGuard.ToolCall.toolCall(
+                "call_1",
+                "shell",
+                "{"command": "shutdown"}"
+            )
+        )
+    )
+);
+```
+
+[1]: https://github.com/DataDog/dd-trace-java/releases/tag/v1.54.0
+{{% /tab %}}
+{{< /tabs >}}
+
+## View AI Guard data in Datadog {#in-datadog}
+
+After your organization's AI Guard feature flag has been enabled and you've instrumented your code using one of the [SDKs](#sdks) (Python, JavaScript, or Java), you can view your data in Datadog on the [AI Guard page][6].
+
+<div class="alert alert-info">You can't see data in Datadog for evaluations performed directly using the REST API.</div>
+
+## Monitoring {#monitoring}
+
+AI Guard includes a built-in dashboard designed to monitor tool evaluations. Datadog can share a dashboard JSON file that you can [import][8] as required.
+
+To ensure that evaluations triggered using the API are displayed on the dashboard, you must manually instrument custom spans using the `ddtrace` library. This setup allows for detailed evaluation tracking and analysis, helping you better understand tool behavior and performance through the AI Guard dashboard. Here's an example implementation:
+
+```py
+with tracer.trace("ai_guard") as span:
+    result = _call_rest_api()  # REST API call
+
+    attributes = result["data"]["attributes"]
+    span.set_tag("ai_guard.target", "tool")  # Use "prompt" if evaluating a prompt
+    span.set_tag("ai_guard.tool_name", "tool_name")  # Specify the tool name if applicable
+    span.set_tag("ai_guard.action", attributes["action"])
+
+    if "reason" in attributes:
+        span.set_tag("ai_guard.reason", attributes["reason"])
+
+    # Optional metadata: tags starting with ai_guard.meta will appear in the outcome table (e.g. input prompt, tool arguments, etc.)
+    span.set_tag("ai_guard.meta.prompt", "the prompt that triggered the tool execution")
+```
+
+The Python SDK handles this process automatically, eliminating the need to manually create the span.
+
+You can use the `datadog.ai_guard.evaluations` metric to count the evaluations AI Guard performed. This metric is tagged by `action`, `blocking_enabled`, `service`, and `env`.
+
+### Set up Datadog Monitors for alerting {#set-up-datadog-monitors}
+
+To create monitors for alerting at certain thresholds, you can use Datadog Monitors, which is included at no additional charge in the Datadog platform. You can monitor AI Guard evaluations with either APM traces or with metrics. For both types of monitor, you should set your alert conditions, name for the alert, and define notifications; Datadog recommends using Slack.
```

Comment:
you could add a link to Monitors: https://docs.datadoghq.com/monitors/

Also, pretty sure monitoring cost money: https://www.datadoghq.com/pricing/.

You might want to remove `which is included at no additional charge in the Datadog platform` so you don't have to maintain it